### DISCUSSION TITLE
Welcome to python-ffmpeg-video-streaming Discussions!

### DISCUSSION THREAD

**Discussion by aminyazdanpanah** [aminyazdanpanah/python-ffmpeg-video-streaming#55] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by chukxoka4** [comment]
Hi, I'm @chukxoka4, a developer myself. However I am looking for a Python Developer from Romania to fill a Python Full Stack and Python/Scala Backend Developer role. I was hoping I would be able to connect with developers interested in this type of opportunity.

**Reply by ahmed-trailfive** [reply]
@chukxoka4 Hi can you share your email. I will share my profile.

**Comment by mangeshchavan** [comment]
how to give input file of subfolder  for ffmpeg.input() on s3 content
eg.
File localtion at s3 bucket is : 
abc/video/vid.mp4

video = ffmpeg_streaming.input(s3, bucket_name=SOURCE_BUCKET_NAME, key=key_value)

**Reply by aminyazdanpanah** [reply]
Try this:
``` python
video = ffmpeg_streaming.input(s3, bucket_name=SOURCE_BUCKET_NAME, key="abc/video/vid.mp4")
```

**Reply by mangeshchavan** [reply]
yes, its working thank you.

**Comment by Gaalen69** [comment]
Might be a silly question, but having to transcode a lot of 4k video for streaming from my Roku channel that I am building. Right now the script is going to take approximately 3.5 hours to transcode 1.18 gb of 4k video to 4k and 1080p HLS. Hoping to speed up the process as I have to transcode about nearly a hundred 4k files. Does the script using hardware acceleration like Nvidia NVENC or is it just using the CPU? Again, probably a silly question, just trying to speed the process up if I can. Let me know what I might do. Thanks, it works great!

**Reply by lakshmansha** [reply]
Hi @Gaalen69 
You can speed up the transcode using Nvidia, I used Tesla T4 GPUs. I tried transcoding 1.5 GB 4K Video in all formats, but it took me approximately 5-10 minutes. For only 1080p, it will take around 4-5 minutes.

Try using the below code,

```python
opts = {
  'hide_banner': None,
  'y': None,
  'vsync': 0,
  'hwaccel': 'cuvid',
  'c:v': 'h264_cuvid'
} 

video = ffmpeg_streaming.input(input_path, pre_opts=opts)
hls = video.hls(Formats.h264("h264_nvenc"),hls_list_size=0, hls_time=2, hls_flags='discont_start+split_by_time')

```

**Comment by noelyeung** [comment]
Hello, my video has different language audio tracks, for example:
```
video.mp4
video_en.mp3
video_jp.mp3
video_de.mp3
```
Is there a way to integrate them into one master playlist?

**Comment by muzahidulislamhadi** [comment]
@aminyazdanpanah Hi, I’m interested in Django/React Video Streaming project listed on QuasarStream. I attempted to reach out on the website but haven’t been successful.

**Reply by mangeshchavan** [reply]
I will help you for django ffmpeg

**Reply by muzahidulislamhadi** [reply]
> I will help you for django ffmpeg

Is the Video Streaming Platform - Basic Web App (https://www.quasarstream.com/video-streaming-basic) available now? Web links are broken.

**Reply by muzahidulislamhadi** [reply]
> I will help you for django ffmpeg

Kindly lemme know.

**Reply by aminyazdanpanah** [reply]
Hi @muzahidulislamhadi,

I appreciate your interest in our packages.

The web link is correct. However, please note that the website is currently undergoing maintenance, and we have temporarily disabled certain features, including payment processing. 

![Screenshotjpg](https://github.com/user-attachments/assets/3da1e64b-ffe5-4389-be5e-a30b21ecbb32)


We are working diligently to launch our updated website in the next few days, possibly by September 2nd.

Thanks for being patient and understanding.

**Reply by muzahidulislamhadi** [reply]
@aminyazdanpanah, Thank you for the response. 
How can i purchase the pro version of this project? Also any demo available?

**Reply by aminyazdanpanah** [reply]
@muzahidulislamhadi, You're welcome 😊

To access the pro version, please contact us and provide a detailed explanation of your requirements. Visit the [Pricing page](https://www.quasarstream.com/pricing) for more information. 

We offer demos for both the basic and pro versions. Check out the basic demo using the following link: https://www.quasarstream.com/vs-demo. 
Please note that this demo build is from a few months ago, but we have updated the basic version with some minor improvements and bug fixes. We may update this demo in a few days.

To view the pro version demo, please schedule a free demo with us using the following link: https://www.quasarstream.com/book-demo.

Please note that the Pro version is intended for company use only. You must be representing a registered company, as unfortunately, we are unable to sell the Pro version to individuals.

**Reply by muzahidulislamhadi** [reply]
@aminyazdanpanah, Demo booking for PRO Version is not working. can i reach out to you on telegram regarding this?

**Reply by aminyazdanpanah** [reply]
Unfortunately, we can only communicate with clients via our website forums, and I am unable to do this through social media.

I hope you understand our limitations and can assist you.

**Comment by muzahidulislamhadi** [comment]
@aminyazdanpanah, Website forums aren't working.

**Comment by juniorcines** [comment]
Hi, is it possible to use without multi-quality, and if the video codec is H264, use copy?

to convert it to HLS faster.
